packages/core/src/DreifussWysiwygEditor.tsx

```diff
@@ -75,10 +78,14 @@ export default function DreifussWysiwygEditor(props: EditorProps) {
           width: 'inherit'
         }
       : {}
-    // TODO: Should be moved to font color plugin
-    // renderLeaf
   }
 
+  const charCount = useCharacterCount()
```

Comment:
if you call it `useCharacterCount()` it sounds like a React hook. But as far as i can tell it's not. Either rename it or make it a proper React hook.